src/cmd-buildextend-metal

```diff
@@ -202,31 +202,6 @@ if [ "${image_type}" == metal4k ]; then
 fi
 
 set -x
-# Extract the target kernel config, which may inform how we build disks.
-kconfig="${tmp_builddir:-tmp}/target-kernel.config"
-target_moduledir=$(ostree --repo="${ostree_repo}" ls "${commit}" /usr/lib/modules | grep -o '/usr/lib/modules/.*')
-ostree --repo="${ostree_repo}" cat "${commit}" "${target_moduledir}/config" > "${kconfig}"
-# Part of: https://github.com/ostreedev/ostree/pull/1959
-# We need to support kernels that don't have this enabled yet, including RHEL8
-# and current Fedora 31.
-# https://bugzilla.redhat.com/show_bug.cgi?id=1765933
-# For a while this was enabled by default, but we backed it out due to the use
-# case of running coreos-installer from old systems (e.g. Debian stable), or
-# ones without CONFIG_FS_VERITY.
-case "${bootfs_type}" in
-  ext4verity)
-        if grep -Eq '^CONFIG_FS_VERITY=y' "${kconfig}"; then
-            disk_args+=("--boot-verity")
-        else
-            echo 'error: Missing CONFIG_FS_VERITY from target kernel config' 1>&2
-            exit 1
-        fi
-        ;;
-  ext4) ;;
-  *) echo "Unsupported bootfs: ${bootfs_type}" 1>&2; exit 1;;
-esac
-rm -f "${kconfig}"
```

Comment:
(I don't think we need to carry this kernel config validation code anymore, the Fedora kernel has this enabled for a while now)